lib/project_types/extension/models/specification_handlers/beacon_extension.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+require "base64"
+require "fileutils"
+require "json"
+
+module Extension
+  module Models
+    module SpecificationHandlers
+      class BeaconExtension < Default
+        SCRIPT_FILE = "beacon_extension.js"
```

Comment:
I imagine this will change based on the feedback of needing a build down below, but I'd recommend against this file naming as the entry into the extension — snake case is not a popular naming format for JavaScript files.

Reply:
updated